### DISCUSSION TITLE
Show skin elements in settings screen

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#15509] [Ideas] [answered]
Right now it is only showing a drop down menu with the names.
![dtauf9](https://user-images.githubusercontent.com/22381687/140637460-37293644-d4f3-4981-9618-05dcdf760b92.png)
I think it would be better to show some elements like the cursor, circles etc. similar to how it is done in stable.
![skydyf](https://user-images.githubusercontent.com/22381687/140637473-86e7f98b-ad7f-4251-ba44-31c641ccbf14.png)

**Comment by peppy** [comment, marked as answer]
A full skinning screen is planned to replace this.

**Reply by respektive** [reply]
alright, that's awesome!
